src/app/features/skills/skill-routing.module.ts

```diff
@@ -0,0 +1,25 @@
+import { NgModule } from '@angular/core'
+import { Routes, RouterModule } from '@angular/router'
+import { SkillProfileComponent } from './skill-profile/skill-profile.component'
+import { SkillListComponent } from './skill-list/skill-list.component'
+
+const routes: Routes = [
+  {
+    path: '', component: SkillListComponent
+  },
+  {
+    path: 'skill-profile/:id', component: SkillProfileComponent
```

Comment:
look like we do a lot of tasks in this pull requests with multiple components. could you please separate to smaller pull request?